src/msr-index.h

```diff
@@ -0,0 +1,74 @@
+/*
```

Comment:
Please add a license header, presumably GPL since this is from the kernel.

Reply:
Sorry, didn't realize it was missing
I can't include a Copyright statement though, as there is none in the sources...